lektor/cli.py

```diff
@@ -285,14 +288,15 @@ def deploy_cmd(ctx, server, output_path, **credentials):
         server_info = config.get_server(server)
         if server_info is None:
             raise click.BadParameter('Server "%s" does not exist.' % server,
-                                     param_hint='server')
-
+                                     param_hint='server')    
```

Comment:
I don't think you need this trailing whitespace :) You might want to configure your text editor to automatically remove it.